src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2016-following-fast-tracks.md

```diff
@@ -473,8 +473,20 @@ Elasticsearch 5.6.x is required. Follow those necessary steps to upgrade:
 
 ##### Upgrade Elasticsearch Version
 
-In order to upgrade your cluster to 5.6.x, please follow the [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html).
-This will require a [full cluster restart upgrade](https://www.elastic.co/guide/en/elasticsearch/reference/current/restart-upgrade.html).
+If your index have been created with LTS 2016 they are in Elasticsearch 2.x format and can be read by Elasticsearch 5.6,
+in this case follow the [full cluster restart upgrade procedure](https://www.elastic.co/guide/en/elasticsearch/reference/current/restart-upgrade.html).
+
+If your index have been created before LTS 2016 they are in Elasticsearch 1.x format and Elasticsearch 5.x will not start,
+in this case some index need to be migrated to the new Elasticsearch 5.x format:
```

Comment:
plural of index is probably "indexes"

Reply:
right, or may be indices as it is used in the doc and api (/_cat/indices)